apps/api/src/scraper/WebScraper/crawler.ts

```diff
@@ -134,6 +134,14 @@ export class WebCrawler {
     limit: number = 10000,
     maxDepth: number = 10
   ): Promise<{ url: string, html: string }[]> {
+    const initialUrl = this.initialUrl
+    this.initialUrl = await checksRedirect(this.initialUrl);
+    if (initialUrl != this.initialUrl) {
```

Comment:
could this initialUrl be ever different from this.initialUrl due to http parsing etc?

Reply:
@nickscamara I added "normalizeUrl"